module_1/first-react-app/src/components/MovieGenre.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useRef } from 'react';
+
+
+/**
+ * 
+ * Requirements
+ * ----------------
+ * implement a component that renders a list of movie genres with currently selected genre highlighted.
+ * The component should accept three properties:
+ * 1. A list of genre names to display. Use the incoming list to render genre buttons.
+ * 2. A name of currently selected genre. Use the name to identify which button to highlight.
+ * 3. A "onSelect" callback property. Call the callback function when the user clicks on any genre button. Pass respective genre name to the callback arguments.
+ *
+ *
+ * @param {*} param0 
+ * @returns 
+ */
+
+const MovieGenre = ({ genres, selectedGenre, onSelect }) => {
+  // Initialize tabs when the component mounts
+  const tabsRef = useRef(null);
+
+  useEffect(() => {
+    // Initialize Materialize tabs component using the ref
+    const instance = window.M.Tabs.init(tabsRef.current);
+    // Cleanup on component unmount
+    return () => {
+      instance.destroy();
+    };
+  }, []);
+
+
+  const customTabsStyle = {
+    height: '50px',
+    lineHeight: '20px',
+    width: '40%', 
+    backgroundColor: '#3e2723',
+    color: '#18ffff', 
+  };
+
+
+  return (
+    <div className="genre-list">
+      <div className="row">
+        <div className="col s12">
+          <ul className="tabs  z-depth-5" style={customTabsStyle} ref={tabsRef}>
+            {genres.map((genre) => (
+              <li key={genre} className="tab">
+                <a href={`#${genre}`} className={genre === selectedGenre ? 'active' : ''} onClick={() => onSelect(genre)}>
```

Comment:
`#${genre}` - please put into a separate const
`genre === selectedGenre ? 'active' : '' ` - please put into a separate const

Reply:
Have introduced const for these in commit : c7a4e1e3bff35922c071c59fe3905a3da37fefef